.gitignore

```diff
@@ -3,7 +3,11 @@ dist
 bundle-report.html
 *.generated*
 .now
+
+# VS Code stuff
 .vscode
+*.code-workspace
```

Comment:
lets share the code workspace, this way we can share common extensions and settings (like prettier formatting)